packages/client/tests/functional/issues/8832/tests.ts

```diff
@@ -1,207 +0,0 @@
-      })
-
-      test('should succeed when "include" involves 32767 records', async () => {
-        const n = 32767
-        await createTags(n)
-        const tags = await getTagsFindManyInclude()
-
-        expect(tags.length).toBe(n)
-      })
-
-      test('should succeed when "in" has 32768 ids', async () => {
-        const n = 32768
-        const ids = await createTags(n)
-        const tags = await getTagsFindManyIn(ids)
-
-        expect(tags.length).toBe(n)
-      })
-
-      test('should succeed when "include" involves 32768 records', async () => {
-        const n = 32768
-        await createTags(n)
-        const tags = await getTagsFindManyInclude()
-
-        expect(tags.length).toBe(n)
-      })
-
-      test('should succeed when "in" has 65536 ids', async () => {
-        const n = 65536
-        const ids = await createTags(n)
-        const tags = await getTagsFindManyIn(ids)
-
-        expect(tags.length).toBe(n)
-      })
-
-      test('should succeed when "include" involves 65536 records', async () => {
-        const n = 65536
-        await createTags(n)
-        const tags = await getTagsFindManyInclude()
-
-        expect(tags.length).toBe(n)
-      })
-
-      /**
-       * Highlight scenarios of the query chunking logic behavior depending on how the bind variables are counted
-       * in the query filters.
-       */
-      describe('unhandled filters', () => {
-        // TODO this test does not fail as expected
-        skipTestIf(providerFlavor === ProviderFlavors.JS_NEON || providerFlavor === ProviderFlavors.JS_PG)(
-          'should fail with `Assertion violation` when "in" is repeated at least twice and "n" is the default value of $QUERY_BATCH_SIZE',
-          async () => {
-            expect.assertions(3)
-            const n = 32766
-            const ids = await createTags(n)
-
-            try {
-              await prisma.tag.findMany({
-                where: {
-                  OR: [
-                    {
-                      id: { in: ids },
-                    },
-                    {
-                      id: { in: ids },
-                    },
-                  ],
-                },
-              })
-            } catch (e) {
-              const error = e as Prisma.PrismaClientKnownRequestError
-              expect(error.message).toContain(
-                'Assertion violation on the database: `too many bind variables in prepared statement, expected maximum of 32767, received 65533`',
-              )
-              expect(error.code).toBe('P2035')
-              expect(error.meta).toMatchObject({
-                database_error:
-                  'too many bind variables in prepared statement, expected maximum of 32767, received 65533',
-              })
-            }
-          },
-        )
-
-        // TODO this test does not fail as expected
-        skipTestIf(providerFlavor === ProviderFlavors.JS_NEON || providerFlavor === ProviderFlavors.JS_PG)(
```

Comment:
This is the important bit that is missing in the new test file, right?

Reply:
This troublesome bit has been reworked so that tests are run on `JS_NEON` / `JS_PG` as well as Rust drivers in the new `chunking-queries` test suite.